### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster-backend#762]
I tried to install Backend using cpanm and I got this:

```
Internal error 002: Unexpected error in the _init_db API call: [Failed to initialize the [SQLite] database backend module: [DB
I connect('dbname=/var/lib/zonemaster/db.sqlite','',...) failed: unable to open database file at /root/.cpanm/work/1621443136.
4007/Zonemaster-Backend-6.1.0/blib/lib/Zonemaster/Backend/DB/SQLite.pm line 37. ]]
Internal error 002
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 2 just after 2.
```

**Comment by pnax** [comment]
Hasn't this been fixed by #764 ?

**Comment by matsduf** [comment]
Fixed by #764.
